vrt/tests.vrt.js

```diff
@@ -76,7 +76,7 @@ describe('visual snapshot tests', () => {
 });
 
 async function snapshot(identifier, viewport = VIEWPORT.desktop) {
-  const image = await page.screenshot();
+  const image = await page.screenshot({captureBeyondViewport: false});
```

Comment:
Just wondering, what inconsistency does this solve? Would this be an issue with playwright as well? Looks like there's no equivalent flag.

Reply:
Related issue: https://github.com/puppeteer/puppeteer/issues/6985

In our project it looks like the page refreshes and the elements are initially rendered incorrectly and the screenshot is taken before the components get updated.  

Playwright doesn't appear to have the same issue, so unless we see any issues come up I would think we'd be fine.

Reply:
👍 Nice job getting to the bottom of these flakes